dlt-database/entity/0001-init_db/Account.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  ManyToOne,
+  JoinColumn,
+  OneToOne,
+  OneToMany,
+  ManyToMany,
+  JoinTable,
+} from 'typeorm'
+import { User } from './User'
+import { Community } from './Community'
+import { TransactionDraft } from './TransactionDraft'
+import { ConfirmedTransaction } from './ConfirmedTransaction'
+
+@Entity('accounts')
+export class Account {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @ManyToOne(() => User, (user) => user.accounts) // Assuming you have a User entity with 'accounts' relation
+  @JoinColumn({ name: 'user_id' })
+  user: User
+
+  // if user id is null, account belongs to community gmw or auf
+  @Column({ name: 'user_id', type: 'int', unsigned: true, nullable: true })
+  userId?: number
+
+  @Column({ name: 'account_nr', type: 'int', unsigned: true, default: 0 })
+  accountNr: number
+
+  @Column({ type: 'binary', length: 32, unique: true })
+  pubkey: Buffer
```

Comment:
No.
This is the user root public key,
and the keys stored in account are the account keys.
For the user from the docu it would be this key derivation path:
```
m/03857ac1'/9cc2'/483e'/8a91'/e5b10f5b8d16'
```
![Derived Keys](https://github.com/gradido/gradido/blob/iota5_dlt-database/docu/Concepts/DLT/derived_keys.md)

Reply:
not bad